static/ansi-to-html.ts

```diff
@@ -392,29 +348,34 @@ function tokenize(text, options, callback) {
         return '';
     }
 
-    function ansiMess(m, g1) {
+    function ansiMess(m: string, g1: string): string {
         ansiMatch = true;
         if (g1.trim().length === 0) {
             g1 = '0';
         }
 
-        g1 = g1.replace(/;+$/, '').split(';');
+        let res: string[] = g1.replace(/;+$/, '').split(';');
 
-        for (var o = 0, len = g1.length; o < len; o++) {
-            callback('display', g1[o]);
+        for (let o: number = 0, len: number = res.length; o < len; o++) {
+            callback('display', res[o]);
         }
 
         return '';
     }
 
-    function realText(m) {
+    function realText(m: string): string {
         callback('text', m);
 
         return '';
     }
 
+    type Token = {
+        pattern: RegExp,
+        sub: (m: string, ...args: any[]) => string
```

Comment:
Code style: extra comma, and missing semicolons after each property definition

Reply:
So replace the commas with semicolons? Like this (1):
```ts
pattern: RegExp;
sub: (m: string, ...args: any[]) => string;
```

or add an extra comma (2):
```ts
pattern: RegExp,
sub: (m: string, ...args: any[]) => string,
```

Reply:
(1) would be consistent with our other type definitions.